Submissions/rufilboy.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "[ILYAS RUFAI]",
```

Comment:
Please remove the square brackets in your submission.

Reply:
Okay

Reply:
> Please remove the square brackets in your submission.

Done